src/bp/ui-admin/src/Pages/Workspace/Bots/BotItemPipeline.tsx

```diff
@@ -140,6 +152,16 @@ const BotItemPipeline: FC<Props> = ({
             </Tag>
           </Tooltip>
         )}
+        {requiresApproval() && (
+          <div className="stage-approval-btns">
+            <Button onClick={rejectStagePromotion} small intent="danger">
```

Comment:
can you use `minimal` attr as well . I feel like until we have a design for this, I prefer we use as less intrusive elements as possible